Sources/MediaCategories/MediaCategoryViewController.swift

```diff
@@ -443,16 +443,23 @@ private extension VLCMediaCategoryViewController {
 
 extension VLCMediaCategoryViewController {
 
-    func play(media: VLCMLMedia) {
-        VLCPlaybackController.sharedInstance().fullscreenSessionRequested = media.subtype() != .albumTrack
```

Comment:
this here shouldn't be removed ^

Reply:
You're right, but I will change the detection to `media.type() != .audio` which makes it easier to read and understand.

I was a bit confused of why it was only albumTrack at first.

Reply:
test it though because I think it wasn't always working